### PR TITLE
III-6036 Add paragraph for completeness sorting

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added paragraph for completeness sorting

---
Ticket: https://jira.uitdatabank.be/browse/III-6036

### CODE REVIEW

File: projects/uitdatabank/docs/search-api/sorting.md [hunk 1]
```diff
@@ -71,17 +72,23 @@ GET /events/?sort[modified]=desc
 
 The events that were modified recently will appear first.
 
-### score
+### completeness
 
-Sorts the data by its relevance. This is the default sort that is used. For an in-depth look how relevance (`score`) is calculated, see the [elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/guide/current/relevance-intro.html).
+Sorts the data by its completeness. This is a score that is calculated based on the amount of fields that are filled in. The more fields are filled in, the higher the completeness score.
```
Comment (JonasVHG) [inline projects/uitdatabank/docs/search-api/sorting.md:77]:
```suggestion
Sorts the data by its completeness. This value is calculated based on the amount of fields that are filled in. The more fields are filled in, the higher the completeness score.
```
Using `score`, when the next paragraph is `score` might be confusing.

**Review by JonasVHG — APPROVED** [review]

File: projects/uitdatabank/docs/search-api/sorting.md [hunk 2]
```diff
@@ -71,17 +72,23 @@ GET /events/?sort[modified]=desc
...
 
 **Examples**
```
Comment (corneelwille) [inline projects/uitdatabank/docs/search-api/sorting.md:79]:
```suggestion
**Example**
```

File: projects/uitdatabank/docs/search-api/sorting.md [hunk 3]
```diff
@@ -71,17 +72,23 @@ GET /events/?sort[modified]=desc
...
 ```
 
-The events with the highest score will appear first.
+The events with the highest completeness score will appear first.
+
+### score
+
+Sorts the data by its relevance. This is the default sort that is used. For an in-depth look how relevance (`score`) is calculated, see the [elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/guide/current/relevance-intro.html).
+
+**Examples**
```
Comment (corneelwille) [inline projects/uitdatabank/docs/search-api/sorting.md:91]:
```suggestion
**Example**
```

File: projects/uitdatabank/docs/search-api/sorting.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ The UiTdatabank Search API provides several options to sort events, places and o
 * `created`
 * `distance`
 * `modified`
+* `completeness`
```
Comment (corneelwille) [inline projects/uitdatabank/docs/search-api/sorting.md:9]: The sorting parameters are also ordered alphabetically, so I would move this to line 5.

File: projects/uitdatabank/docs/search-api/sorting.md [hunk 5]
```diff
@@ -71,17 +72,23 @@ GET /events/?sort[modified]=desc
 
 The events that were modified recently will appear first.
 
-### score
+### completeness
```
Comment (corneelwille) [inline projects/uitdatabank/docs/search-api/sorting.md:75]: To be consistent with the now alphabetical order, this block should be moved between the sorting parameter `availableTo` and `created`

**Review by corneelwille — CHANGES_REQUESTED** [review]
Same (minor) feedback regarding the sorting as for the advanced query field

**Pull request merged**